contribs/gmf/src/directives/layertree.js

```diff
@@ -202,11 +202,12 @@ gmf.LayertreeController = function($http, $sce, $scope, ngeoCreatePopup,
  * @param {GmfThemesNode} node Layer tree node.
  * @param {number=} opt_depth ngeo layertree node depth.
  * @param {boolean=} opt_createWMS True to allow create wms layer.
+ * @param {GmfThemesNode=} parentNode parent node of the first parameter.
  * @return {ol.layer.Base} The OpenLayers layer or group for the node.
  * @export
  */
 gmf.LayertreeController.prototype.getLayer = function(node, opt_depth,
-        opt_createWMS) {
+        opt_createWMS, parentNode) {
```

Comment:
The parentNode should come from the controller, when the main ngeo directive calls the template.
This should be modified in https://github.com/camptocamp/ngeo/blob/master/src/directives/layertree.js#L211
Do you agree @ger-benjamin